bin/diff-check: No such file or directory

Found this seemingly very useful action, but when trying to run it I got:

```
Run bin/diff-check >> $GITHUB_STEP_SUMMARY
  bin/diff-check >> $GITHUB_STEP_SUMMARY
  shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
  env:
    pythonLocation: /opt/hostedtoolcache/Python/3.10.14/x64
    PKG_CONFIG_PATH: /opt/hostedtoolcache/Python/3.10.14/x64/lib/pkgconfig
    Python_ROOT_DIR: /opt/hostedtoolcache/Python/3.10.14/x64
    Python2_ROOT_DIR: /opt/hostedtoolcache/Python/3.10.14/x64
    Python3_ROOT_DIR: /opt/hostedtoolcache/Python/3.10.14/x64
    LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.10.14/x64/lib
/home/runner/work/_temp/[47](https://github.com/apytypes/apytypes/actions/runs/9329981048/job/25683042789?pr=457#step:6:50)6019dd-209a-4187-97ef-7edb2a35c12c.sh: line 1: bin/diff-check: No such file or directory
Error: Process completed with exit code 127.
```
so my interpretation is that `bin/diff-check` cannot be accessed from my workflow?

--- a/action.yml
+++ b/action.yml
@@ -14,11 +14,7 @@ runs:
     - run: ${{ inputs.command }}
       shell: bash
 
-    - name: Add Action to $GITHUB_PATH
-      run: echo "$GITHUB_ACTION_PATH" >> $GITHUB_PATH
+    - run: $GITHUB_ACTION_PATH/bin/diff-check >> $GITHUB_STEP_SUMMARY
       shell: bash
       env:
         GITHUB_ACTION_PATH: ${{ github.action_path }}
-
-    - run: bin/diff-check >> $GITHUB_STEP_SUMMARY
-      shell: bash

